### PR TITLE
Automated Test - COMMERCE-6106 | master

### PR DESCRIPTION
**Pull request by ewertonluna**

### CODE REVIEW

**Comment by ewertonluna** [comment]
https://issues.liferay.com/browse/COMMERCE-6106, cc: @PietroBocale

**Comment by Pietro69** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro [hunk 1]
```diff
@@ -22,4 +22,41 @@ definition {
...
+
+		Type(
+			locator1 = "TextInput#ANY",
+			key_text = "Client Secret",
+			value1 = "PayPal Client Secret");
+
+		Select(
+				key_fieldLabel = "Mode",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "Sandbox");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro]: Can we use a variable? in this way we can use this macro in the future not only for Sandbox mode

**Review by Pietro69 — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro (hunk 1 above)
Comment (ewertonluna) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro]: Fixed.

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
Hi @ewertonluna, please fix your source formatting. Thanks!

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Pull request closed without merging**